### PR TITLE
Making the large code snippets scrollable

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) brings some improvements to large code snippets, and make them scrollable. 
It only applies to those blocks, that are directly shown in page, and not to those that are used in a collapsible block.

### Before

Xnip2020-03-30_09-29-07

### After

Xnip2020-03-30_09-28-48

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/async-operations.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=#using-deferredinterface-for-background-operations
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/async-operations.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=#using-deferredinterface-for-deferred-operations
- https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/create-integration-with-api.html

## Links to Magento source code

N/A

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/common/css/devdocs.css [hunk 1]
```diff
@@ -36,3 +36,10 @@
   font-size: 1.3em;
   margin: 1.5em 0 0.5em 0;
 }
+
+/* Makes all the big code snippets scrollable */
+.content-wrap > div[class*="language-"].highlighter-rouge .highlight {
```
Comment (belbiy) [inline src/common/css/devdocs.css]: This is probably over-specific selector, but it's okay. I will move this functionality to the theme (https://github.com/magento-devdocs/devdocs-theme/) and will make sure we use generic selector.

File: src/common/css/devdocs.css [hunk 2]
```diff
@@ -36,3 +36,10 @@
...
+  overflow: scroll;
```
[the remark this answers is not in the document]
Comment (eduard13) [reply src/common/css/devdocs.css]: Sure, makes sense.

File: src/common/css/devdocs.css (hunk 1 above)
Comment (eduard13) [reply src/common/css/devdocs.css]: I wasn't aware of this project, however it seems that `.highlighter-rouge` isn't needed here, but wasn't sure about that.

**Comment by eduard13** [comment]
Hi @belbiy, thank you for the suggestion, could you please check the updated version?

**Comment by dobooth** [comment]
running tests

**Pull request merged**